tests/integration/config/test_character_configuration.py

```diff
@@ -48,13 +44,13 @@
 characters = (Alice, Bob, Ursula)
 
 # Auxiliary Support
-blockchain_only_configurations = (FelixConfiguration, StakeHolderConfiguration)
-blockchain_only_characters = (Felix, StakeHolder)
+blockchain_only_configurations = (StakeHolderConfiguration, )
+blockchain_only_characters = (StakeHolder, )
 
 # Assemble
 characters_and_configurations = list(zip(characters, configurations))
-all_characters = tuple(characters + blockchain_only_characters)
-all_configurations = tuple(configurations + blockchain_only_configurations)
+all_characters = tuple(characters, )
+all_configurations = tuple(configurations, )
```

Comment:
Are we no longer including `blockchain_only_characters`(i.e. only `StakeHolder` character) and the associated `blockchain_only_configurations`(i.e. only`StakeHolderConfiguration`)?

I know we are going to get rid of it eventually - but just checking if we wanted to keep it for now.

Reply:
Since it's EOL I preemptively removed it here, albeit eagerly.  I think the remaining tuple was empty so it was awkward to leave it dangling.

Reply:
👍